README.md

```diff
@@ -1,11 +1,103 @@
-# Project
+# Private Preview: AutoML Model Testing
 
-As the maintainer of this project, please make a few updates:
+> Welcome to the PRIVATE PREVIEW of **Azure Machine Learning Automated ML Model Testing**.
+> The AutoML model testing feature provides support for automated model testing.
+> This is an early PREVIEW, still not announced and not supported publicly so
+> it's defined as PRIVATE from that point of view, although anyone can access to it.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Automated Machine Learning, also referred to as Automated ML or AutoML, is
+the process of automating the time consuming, iterative tasks of machine
+learning model development. It allows data scientists, analysts, and
+developers to quickly and easily build ML models with high scale, efficiency,
+and productivity all while sustaining model quality.
+
+The AutoML model testing feature allows the user to test the models which are
+generated by AutoML training runs. Testing a model takes as input a test dataset
+and outputs predictions and metrics for those predictions.
+There are currently two supported scenarios:
+
+1. Test the best model which is created by the main AutoML run.
+   The model testing run will be started automatically at the end of the main run
+   after all models have been created and the best model has been identified.
+2. Test any model after the main AutoML run has completed ("on-demand" model testing).
+   These test runs are started manually by the user either using the SDK or the UI.
+
+Model testing is supported for the following task types:
+
+- Classification
+- Regression
+- Forecasting
+
+There are three ways to start a test run from a script or a notebook.
```

Comment:
Please add a title here like "Quick Walkthrough" or "Getting Started".

But before this section, is there anything about SETUP that a user needs to do?

For instance, requirements, version needed, etc. any pre-requisite should come first, before the "Getting Started".

"Limitations" is okay to have it later, as you have it.
But we need the "pre-requisites" up-front with a Title like "Requirements" or "Pre-requisites" and then a few lines explaining it.  :)
Even related to the Azure Subscription, Workspace requisite, etc. Anything that is needed before trying this feature. :)